src/yunohost/domain.py

```diff
@@ -61,7 +61,11 @@ def domain_list(exclude_subdomains=False):
                 continue
         result_list.append(domain)
 
-    return {'domains': result_list}
+    result_list.sort()
```

Comment:
You can use `sorted(result_list)` bellow to save a line if you want (but that won't be a in-place sort)

Reply:
I think I prefer as it is, because result_list is used twice below (and we don't want sort() to run twice, do we ? ;) )

Reply:
```suggestion
```

Domain sorting is to be addressed by https://github.com/YunoHost/yunohost/pull/860